<!doctype html>
<html lang="en">

	<head>
		<meta charset="utf-8">

		<title>Open Tree of Life</title>

		<meta name="description" content="Slides for Duke SDG talk">
		<meta name="author" content="Karen Cranston">

		<meta name="apple-mobile-web-app-capable" content="yes" />
		<meta name="apple-mobile-web-app-status-bar-style" content="black-translucent" />

		<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, minimal-ui">

		<link rel="stylesheet" href="css/reveal.css">
		<!-- Change theme here -->
		<link rel="stylesheet" href="css/theme/solarized.css" id="theme">

		<!-- Code syntax highlighting -->
		<link rel="stylesheet" href="lib/css/zenburn.css">

		<!-- Printing and PDF exports -->
		<script>
			var link = document.createElement( 'link' );
			link.rel = 'stylesheet';
			link.type = 'text/css';
			link.href = window.location.search.match( /print-pdf/gi ) ? 'css/print/pdf.css' : 'css/print/paper.css';
			document.getElementsByTagName( 'head' )[0].appendChild( link );
		</script>

		<!--[if lt IE 9]>
		<script src="lib/js/html5shiv.js"></script>
		<![endif]-->
	</head>

	<body>

		<div class="reveal">

			<!-- Any section element inside of this container is displayed as a slide -->
			<div class="slides">
				<section id="titleSlide">
					<img src="assets/OpenTree_leaf_logo.png">
					<h3>A comprehensive, online, updateable tree of life</h3>
					<p>
						<small><a href="https://github.com/kcranston/OpenTree_NSF_2016">https://github.com/kcranston/OpenTree_NSF_2016</a></small>
					</p>
				</section>

				<section>
					<h1>So, where is the tree of life?</h1>
				</section>

				<section>
					<p>
						<img width="800" src="assets/atols.png">
					</p>
				</section>

				<section>
					<p>
						<img src="assets/scopus_phylogeny.png">
					</p>
				</section>

				<section>
					<h2>What do we know about the tree?</h2>
					<h2>Can I browse / search / download it?</h2>
				</section>

				<section>
					<p>
						Summarize existing phylogenetic knowledge into a tree of life that
						is 1. Complete; 2. Online; 3. Updateable.
					</p>
					<p>
						<img width="500" src="assets/circleTreeRedBlue.png">
					</p>
				</section>

				<section>
						<p>
							<img src="assets/overview.png">
						</p>
				</section>

				<section>
					<blockquote>
						Phylogeny provides a mechanism through which to interpret the
						patterns and processes of evolution and to predict the responses of
						life to rapid environmental change. Phylogenies and phylogenetic
						methods are now being used to enhance agriculture, identify and
						combat diseases, conserve biodiversity, and predict responses to
						global climate change and to biological invasions.
					</blockquote>
					<dt>(tl;dr: We need trees to do science)<dt>
					<!-- <p><small>*Open Tree grant proposal</p> -->
				</section>

				<section>
					<h1>Status</h1>
				</section>

				<section id="people">
					<p>Original funding (2011 - 2013): 11 PIs, 10 institutions</p>
					<p>Supplement (2013 - present): Cranston, Holder, Smith</p>
					<p>
					<div class="ctr w50">
	    			<img width="800" src="assets/opentree_people.png">
	  			</div>
				</p>
				</section>

				<section>
						<p>
							<img src="assets/overview.png">
						</p>
				</section>

				<section>
					<h2>Two Greatest Challenges</h2>
					<h3>(Technical and social)</h3>
					<p>I. Publishing tree files not a community norm</p>
				</section>

				<section>
						<p>
							<img width="800" src="assets/fractionTreesArchived.png">
						</p>
				</section>

				<section>
					<h2>phylesystem</h2>
					<p>A git-based data store for community-curated phylogenetic estimates. <a href="http://dx.doi.org/10.1093/bioinformatics/btv276">Bioinformatics (2015) 31 (17): 2794-2800</a></p>
					<ul>
						<li>7757 trees from 3401 studies</li>
						<li>4779 commits from 117 curators (90 non-opentree)</li>
						<li>git backend + python API + curation application</li>
					</ul>
				</section>

				<section>
					<h2>Biggest challenges</h2>
					<h3>(Technical and social)</h3>
					<p>I. Publishing tree files not a community norm</p>
					<p>II. Taxonomy resources are incomplete and out of date</p>
				</section>

				<section>
					<h2>Taxonomy woes</h2>
					<p class="fragment">Many taxonomy databases; differ in coverage and fitness</p>
					<p class="fragment">Long lag taxonomy publication -> taxonomy databases</p>
					<p class="fragment">Taxa and names missing</p>
					<p class="fragment">Relationships out of date</p>
					<p class="fragment">Much data not openly available</p>
				</section>

				<section>
					<h2>Open Tree Taxonomy</h2>
					<p>
						<img  width="800" src="assets/ott_stats.png">
					</p>
					<ul>
						<li>7 input taxonomies + user-contributed patches</li>
						<li>clade-based input priority</li>
						<li>TNRS services to resolve synonyms, homonyms, etc</li>
					</ul>
				</section>

				<section>
					<p>
						<img  width="850" src="assets/PNAS.png">
					</p>
					<p>
						<img  width="650" src="assets/synth_stats.png">
					</p>
				</section>

				<!-- <section>
						<p>
							<img  width="800" src="assets/sciam_circle_life.jpg">
						</p>
				</section> -->

				<section>
						<p>
							<img src="assets/impactStory_PNAS.png">
						</p>
				</section>

				<section>
					<h1>Infrastructure</h1>
				</section>

				<section>
					<h2>Infrastructure</h2>
					<p><strong>bold</strong> = version 2.0</p>
					<ul>
						<li>Web applications:</li>
							<ul>
								<li><dt>study curator</dt></li>
								<li>browsers for tree and for taxonomy</li>
							</ul>
						<li>Pipelines:</li>
							<ul>
								<li>taxonomy merging</li>
								<li><dt>tree synthesis</dt></li>
							</ul>
						<li>Databases:</li>
							<ul>
								<li><strong>tree store</strong> (github)</li>
								<li>synthetic tree and taxonomy (neo4j)</li>
								<li><strong>tree index</strong> (postgreSQL)</li>
							</ul>
					</ul>
				</section>

				<section>
					<h2>Interoperability via APIs</h2>
						<table>
							<tbody>
								<tr>
									<td style="vertical-align:middle;"><img class="plain" width="200" src="assets/ropensci_main.png"></td>
									<td style="vertical-align:middle;">rotl: R package wraps APIs</td>
								</tr>
								<tr>
									<td style="vertical-align:middle;"><img class="plain" width="250" src="assets/phylotastic.png"></td>
									<td style="vertical-align:middle;">provides trees given species</td>
								</tr>
								<tr>
									<td style="vertical-align:middle;"><img class="plain" width="150" src="assets/arbor.png"></td>
									<td style="vertical-align:middle;">imports trees for comparative analyses</td>
								</tr>
							</tbody>
						</table>
					</section>

					<section>
						<h2>Other collaboration</h2>
							<table>
								<tbody>
									<tr>
										<td style="vertical-align:middle;"><img class="plain" width="200" src="assets/gbif.jpg"></td>
										<td style="vertical-align:middle;">Providing taxonomy feedback</td>
									</tr>
									<tr>
										<td style="vertical-align:middle;"><img class="plain" width="200" src="assets/futurephy.jpg"></td>
										<td style="vertical-align:middle;">Joint workshops, webinars</td>
									</tr>
								</tbody>
							</table>
						</section>

				<section>
					<h1>Community building</h1>
				</section>

				<section>
					<h2>Tree-for-all hackathon</h2>
					<p>U Michigan, September 2014</p>
					<p>
						<a href="https://github.com/OpenTreeOfLife/hackathon">https://github.com/OpenTreeOfLife/hackathon</a>
					</p>
					<ul>
						<li>Invited people to build on our APIs</li>
						<li>Participants from Arbor, rOpenSci, iDigBio, Supertree Toolkit,
							IPNI, Atlas Living Australia, Fossil Calibration Database,
							Species File Group</li>
						</ul>
				</section>

				<section>
					<h2>FuturePhy / Arbor / OpenTree workshop</h2>
					<p>U Florida, February 2016</p>
					<p>
						<a href="https://blog.opentreeoflife.org/2016/03/09/futurephy-clade-workshops/">https://blog.opentreeoflife.org/2016/03/09/futurephy-clade-workshops/</a>
					</p>
					<ul>
						<li>Clades: barnacles, catfishes, beetles</li>
						<li>Participants: taxonomy, systematics, ecology, phylogenetic
							methods, bioinformatics, genomes, ontologies, and
							scientific illustration</li>
						<li>Tested tree input, custom synthesis, conflict visualization</li>
					</ul>
				</section>

				<section>
					<h2>Demo!</h2>
					<p>
						<a href="https://tree.opentreeoflife.org">https://tree.opentreeoflife.org</a>
					</p>
				</section>

				<section>
					<h1>What could we do better?</h2>
				</section>

				<section>
					<h2>Synthetic tree woefully underrepresents phylogenetic knowledge</h2>
						<table>
							<thead>
								<tr>
									<th>Clade</th>
									<th>Tips</th>
									<th>Phylo tips</th>
									<th>% phylo</th>
								</tr>
							</thead>
							<tbody>
								<tr>
									<td>Embryophyta</td>
									<td>296,611</td>
									<td>14,425</td>
									<td>4.9%</td>
								</tr>
								<tr>
									<td>Fungi</td>
									<td>309,631</td>
									<td>628</td>
									<td>0.2%</td>
								</tr>
								<tr>
									<td>Metazoa</td>
									<td>1,467,443</td>
									<td>21,323</td>
									<td>1.5%</td>
								</tr>
								<tr>
									<td>Insecta</td>
									<td>979,709</td>
									<td>3098</td>
									<td>0.3%</td>
								</tr>
							</tbody>
						</table>
					</section>

					<section>
						<h2>Improving the tree</h2>
						<p class="fragment">Incorporate more trees from phylesystem</p>
						<p class="fragment">Import trees from TreeBASE, Dryad</p>
						<p class="fragment">Scrape trees from images</p>
						<p class="fragment">Incorporate more taxonomy feedback / resources</p>
						<p class="fragment">Encourage more community input of trees</p>
					</section>

					<section>
						<h2>OpenTree needs more curators</h2>
						<p>Motivating producers of trees to contribute data</p>
						<ul>
							<li>Social motivation</li>
								<ul>
									<li>curator statistics, leaderboards</li>
									<li>favorites, notifications</li>
								</ul>
							<li>Research services</li>
								<ul>
									<li>custom synthesis</li>
									<li>conflict analysis</li>
									<li>hosting</li>
								</ul>
							</ul>
					</section>

					<section>
						<h2>More needed improvements</h2>
						<p class="fragment">Tree visualization</p>
						<p class="fragment">Branch lengths / divergence times</p>
						<p class="fragment">Links back to raw data (sequences, specimens)</p>
						<p class="fragment">Greatly expanding dark parts of the tree</p>
					</section>

					<section>
						<p>
							<a href="http://www.scientificamerican.com/sciam/assets/Image/B060BD8C-BF9E-4683-8E29AA7B6B3F1689_landingbanner%20(1).jpg?shunter=1457194944393" target="_blank">
							<img width="800" src="assets/sciam_circle_life_text.png"></a>
							<p>Scientific American, March 2016</p>
						</p>
					</section>

					<section>
						<div class="ulist">
							<img src="assets/Hug_otu_mapping.png" width="65%" style="float: left">
							<ul style="width: 30%;">
								<p>Need resources describing OTUs in microbial trees</p>
								<p>(aka Tell Me More about “Prevotella dentalis ES 2772 DSM 3688”)</p>
							</ul>
						</div>
					</section>

					<section>
						<h2>Summary</h2>
						<p>Have process for curating and synthesizing phylogenetic
							and taxonomic data into comprehensive tree of life</p>
						<p>Have data and services available through APIs; in
							use by biodiversity informatics community</p>
						<p>Need to motivate participation of systematics community
						 in curation through tools and services</p>
					</section>

			</div>

		</div>

		<script src="lib/js/head.min.js"></script>
		<script src="js/reveal.js"></script>

		<script>

			// Full list of configuration options available at:
			// https://github.com/hakimel/reveal.js#configuration
			Reveal.initialize({
				controls: true,
				progress: true,
				history: true,
				center: true,
				slideNumber : true,
				transition: 'slide', // none/fade/slide/convex/concave/zoom

				// Optional reveal.js plugins
				dependencies: [
					{ src: 'lib/js/classList.js', condition: function() { return !document.body.classList; } },
					{ src: 'plugin/markdown/marked.js', condition: function() { return !!document.querySelector( '[data-markdown]' ); } },
					{ src: 'plugin/markdown/markdown.js', condition: function() { return !!document.querySelector( '[data-markdown]' ); } },
					{ src: 'plugin/highlight/highlight.js', async: true, condition: function() { return !!document.querySelector( 'pre code' ); }, callback: function() { hljs.initHighlightingOnLoad(); } },
					{ src: 'plugin/zoom-js/zoom.js', async: true },
					{ src: 'plugin/notes/notes.js', async: true }
				]
			});

		</script>

	</body>
</html>
